code/datums/keybinding/xeno.dm

```diff
@@ -1233,33 +1233,33 @@
 	keybind_signal = COMSIG_XENOABILITY_SPIDERLING_MARK
 	hotkey_keys = list("V")
 
-/datum/keybinding/xeno/rewind
-	name = "rewind"
-	full_name = "Wraith: Time Shift"
-	description = "Save the location and status of the target. When the time is up, the target location and status are restored"
-	keybind_signal = COMSIG_XENOABILITY_REWIND
-	hotkey_keys = list("C")
+/datum/keybinding/xeno/wraith_blink
+	name = "Wraith's Blink"
```

Comment:
dont change the name that resets the keybond

Reply:
fixed